pkg/network/driver/common.go

```diff
@@ -161,6 +162,24 @@ func (h *NetworkUtilsHandler) ConfigureIpForwarding(proto iptables.Protocol) err
 	return err
 }
 
+func (h *NetworkUtilsHandler) IsIpv4Enabled(interfaceName string) (bool, error) {
+	link, err := h.LinkByName(interfaceName)
+	if err != nil {
+		return false, err
+	}
+	addrList, err := h.AddrList(link, netlink.FAMILY_V4)
+	if err != nil {
+		return false, err
+	}
+
+	for _, addr := range addrList {
+		if addr.IP.IsGlobalUnicast() {
```

Comment:
Why IPv4 is not enabled if the addres is not globally unique ? Maybe we can change func name to `HasIPv4GlobalUnicastAddress`